### DISCUSSION TITLE
setting the battery cell voltage low & critical to lower values for Li-on battery

### DISCUSSION THREAD

**Discussion by Tranklo** [iNavFlight/OpenTX-Telemetry-Widget#124] [Q&A]
### To reproduce

1. Go to '....'
4. Should it be doing this?

**Comment by cptX** [comment]
Is there an answer to this? I fly my multicopter with li-ions and I get low battery alerts from the beginning. The Tx is vibrating and is very destructing experience, I cannot focus on my flight. How can we change the limits?
